mimic-iii-carevue/install.sh

```diff
@@ -0,0 +1,578 @@
+
+            if [[ "$pg_hba" =~ "pg_hba.conf" ]]; then
+                test_db_login=true
+                echo "数据库配置文件'pg_hba.conf'的路径是$pg_hba"
+
+                if grep -E "$ptn" "$pg_hba"; then
+                    echo -e "===================================="
+                    echo -e "\n"
+                    echo -e "请输入操作系统用户登录密码!"
+                    echo -e "\n"
+                    echo -e "===================================="
+                    pg_hba_dir=$(sudo dirname "$pg_hba")
+
+                    # backup pg_hba.conf
+                    cp "$pg_hba" "$pg_hba_dir"/pg_hba.conf.bk
+                    # 判断是否是Mac OS
+                    os="$(uname)"
+                    sed_version="$(which sed)"
+
+                    # reset the mode of local connection as "trust"
+                    if [[ "$os" == "Darwin" ]] && [[ "$sed_version" == "/usr/bin/sed" ]]; then
+                        sed -i.bk -E "$sed_ptn" "$pg_hba"   # FreeBSD-sed
+                    else
+                        sed -i -E "$sed_ptn" "$pg_hba"   # GNU-sed
+                    fi
+                    echo -e "\n"
+                    echo -e "数据库本地连接模式已重置为'trust'!"
+                    echo -e "\n"
+                    echo -e "===================================="
+                    # restart the database to load the configuration
+                    psql -U postgres -q -c "SELECT pg_reload_conf();"
+                    echo -e "===================================="
+                    echo -e "\n"
+                    echo -e "数据库配置文件'pg_hba.conf'已重新加载生效!"
+                    echo -e "\n"
+                    echo -e "===================================="
+                else
+                    echo -e "===================================="
+                    echo -e "\n"
+                    echo -e "数据库本地连接模式为'trust'!"
+                    echo -e "\n"
+                    echo -e "===================================="
+                fi
+
+            else
+                test_db_login=false
+                echo -e "===================================="
+                echo -e "\n"
+                echo -e "数据库用户'postgres'登录密码输入错误"
+                echo -e "请重新输入数据库用户'postgres'的登录密码!"
+                echo -e "\n"
+                echo -e "===================================="
+            fi
+    done
+fi
+
+
+
+# create database user and database with the same name as the operating system user name
+if [[ $lan_sel == 1 ]]; then
+    # English
+
+    # create database user
+    if psql -U postgres -t -c '\du' | cut -d \| -f 1 | grep -qw "$(whoami)"; then
+        # the database user with the same name as the system user exists
+        echo ""
+    else
+        # the database user with the same name as the system user does not exist
+        echo "--- The database user with the same name as the system user does not exist. ---"
+        echo "--- The database user with the same name as the system user will be created. ---"
```

Comment:
Change this to carry on from the last sentence, e.g. "A new user will be created"